docs/create-packages/Supporting-Multiple-Target-Frameworks.md

```diff
@@ -15,6 +15,32 @@ This article describes the layout of a NuGet package, regardless of how the pack
 
 You must manually lay out the package as described in this article when using the convention-based working directory method described in [Creating a package](../create-packages/creating-a-package.md#from-a-convention-based-working-directory). For an SDK-style project, the automated method is recommended, but you may also choose to manually lay out the package as described in this article.
 
+## Declaring dependencies
```

Comment:
We should mention that packing a project file, NuGet will try to automatically generate the dependencies from the project. The information below about using a nuspec file should only be necessary for advanced scenarios.

Reply:
Thx, updating